http/vertx-web-client/src/test/java/io/quarkus/ts/http/vertx/webclient/PureVertxHttpClientTest.java

```diff
@@ -46,4 +47,10 @@ public void quarkusTestFipsVertxHttpClient() throws InterruptedException {
         done.await(TIMEOUT_SEC, TimeUnit.SECONDS);
         assertEquals(0, done.getCount(), String.format("Missing %d events.", EXPECTED_EVENTS - done.getCount()));
     }
+
+    @AfterEach
+    public void cleanUp(){
+        if (Vertx.vertx() != null)
+            Vertx.vertx().close();
```

Comment:
here you create Vert.x and immediately close it 
![image](https://github.com/quarkus-qe/quarkus-test-suite/assets/43821672/deb330a3-3fcc-4e31-b6f0-03b8e46af3db)

you will need to keep `Vertx.vertx()` in variable (I guess static one :-() used when WebClient is created and close it here. Please also close `WebClient` created in `BeforeEach`.